app/grandchallenge/emails/admin.py

```diff
@@ -0,0 +1,96 @@
+from django.contrib import admin, messages
+from django.db.transaction import on_commit
+from django.utils.timezone import now
+from django.utils.translation import ngettext
+from markdownx.admin import MarkdownxModelAdmin
+
+from grandchallenge.emails.models import Email
+from grandchallenge.emails.tasks import send_bulk_email
+from grandchallenge.emails.utils import SendActionChoices
+
+
+def schedule_emails(modeladmin, queryset, request, action):
+    emails = queryset.filter(sent=False)
+    if emails:
+        for email in emails:
+            send_admin_emails = send_bulk_email.signature(
+                kwargs={
+                    "action": action,
+                    "subject": email.subject,
+                    "body": email.body,
+                },
+                immutable=True,
+            )
+            on_commit(send_admin_emails.apply_async)
+
+        updated = emails.update(sent=True, sent_at=now())

```

Comment:
The guard and update should be in the Celery task, not the admin action as Celery tasks can be run twice (eg, if they are interrupted they will be placed again on another worker).